lib/datahub-client/data_platform_catalogue/client/datahub_client.py

```diff
@@ -275,9 +275,9 @@ def get_table_details(self, urn) -> Table:
             custodians = parse_custodians(response)
             tags = parse_tags(response)
             glossary_terms = parse_glossary_terms(response)
-            created, modified = parse_created_and_modified(properties)
-            modified = parse_last_modified(response)
-            updated = parse_updated(response)
+            created = parse_data_created(properties)
+            modified = parse_metadata_last_ingested(response)
+            updated = parse_last_datajob_run_date(response)
```

Comment:
I think this works at the moment - but could there be tables that we want to get the lastModified date from the property in datahub, like when a table is not produced by dbt?

Reply:
This is one of the bits I think is actually wrong - but I'm saving that for the next PR, this one is just renaming things